buildspec-windows.yml

```diff
@@ -0,0 +1,63 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - choco install cyg-get -y --no-progress
+      - cyg-get bash patch bison flex make wget perl
```

Comment:
Would `curl` also be available? Just curious as I'm wondering whether we can consistently move towards `curl` for all download steps across all platforms.

Reply:
Possibly, trying.